Formula/pyqt@5.rb

```diff
@@ -15,8 +16,9 @@ class PyqtAT5 < Formula
   end
 
   depends_on "pyqt-builder" => :build
+  depends_on "python@3.10"  => :build
+  depends_on "python@3.9"   => :build
```

Comment:
You need one of these to be available during the test, no? Preferably both of them so we can test both.

Reply:
Whoops, they should definitely both be test dependencies.  The whole point of the test is try both!